ee/clickhouse/models/group.py

```diff
@@ -0,0 +1,31 @@
+import datetime
+import json
+from typing import Dict, Optional
+
+from django.utils.timezone import now
+
+from ee.clickhouse.sql.groups import INSERT_GROUP_SQL
+from ee.kafka_client.client import ClickhouseProducer
+from ee.kafka_client.topics import KAFKA_GROUPS
+
+
+def create_group(
+    team_id: int,
+    group_type_index: int,
+    group_key: str,
+    properties: Optional[Dict] = {},
+    timestamp: Optional[datetime.datetime] = None,
+):
+    if not timestamp:
+        timestamp = now()
+
+    data = {
+        "group_type_index": group_type_index,
+        "group_key": group_key,
```

Comment:
group_key is the name of the group?

Reply:
More like the unique identifier.

So from https://posthog.com/docs/user-guides/group-analytics if you did `posthog.groupIdentify('group_type', 'group_key', {})` this is the group_key